CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## [Unreleased] - ReleaseDate
 ### Added
 
+- Add `MntFlags` and `unmount` on all of the BSDs.
+  ([#1849](https://github.com/nix-rust/nix/pull/1849))
+- Added a 'Statfs::flags' method.
+  ([#1849](https://github.com/nix-rust/nix/pull/1849))
+- Added a 'Statfs::flags' method.
```

Comment:
duplicate entries?

Reply:
That's really weird.  Must've happened when I reordered my commits.  And the weirdest part is that I couldn't make it go away by creating a second commit that removed the dupe and combining it as a fixup with the first commit.  But I finally got it fixed.